src/io/flutter/preview/PreviewView.java

```diff
@@ -291,6 +294,15 @@ private void selectPath(TreePath selectionPath, boolean focusEditor) {
       return;
     }
 
+    if (focusEditor) {
```

Comment:
rename both to 'jumpToSource'? 'showSource'?

Reply:
Not sure which "both".
I speculatively changed the name of the parameter and the name of the analytics event.
PTAL

Reply:
Sorry, I meant both `jumpToSourceDartOutline` and `jumpToSourceFlutterOutline` could just be simplified to `jumpToSource` - that we don't need to differentiate between the source element type.